### PR TITLE
Add icons on several files 

### PR DESCRIPTION
**Pull request by Heathercoraje**
relates #75

### CODE REVIEW

File: src/components/help.jsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ const Help = () => (
     <p>You can also click on the LIKE and DISLIKE buttons</p>
     <p>Click LIST to see the jobs that you like</p>
     <p>Click HOME to go back to jobs</p>
+
```
Comment (Karyum) [inline src/components/help.jsx]: what is this empty line for?

File: src/containers/career.jsx [hunk 2]
```diff
@@ -114,16 +116,16 @@ class Career extends Component {
...
+              <Info size='2em'
                 onClick={() => {
                   this.flipCard();
-                }}> Info</button>
+                }}/>
             </div>
             <div className="back">
               <CareerInfo career={this.state.careers[this.state.currentIndex]}/>
-              <button className="btn1" onClick={() => {
+              <Back size='2em'className="btn1" onClick={() => {
```
Comment (Karyum) [inline src/containers/career.jsx]: onClick on a new line.
also the => is a return you dont need the curly braces if you are returning 1 thing

File: src/containers/career.jsx [hunk 3]
```diff
@@ -114,16 +116,16 @@ class Career extends Component {
             onTouchEnd={event => this.handleTouchEnd(event)}>
             <div className="front">
               <CareerCard career={this.state.careers[this.state.currentIndex]}/>
-              <button
+              <Info size='2em'
                 onClick={() => {
```
Comment (Karyum) [inline src/containers/career.jsx]: => = return

File: src/containers/career.jsx (hunk 2 above)
Comment (Heathercoraje) [reply src/containers/career.jsx]: if I remove the curry braces, it says Uncaught ```Error: Module build failed: SyntaxError: Unexpected token, expected } (119:61)```

**Pull request merged**